lms/djangoapps/teams/static/teams/js/views/edit_team_members.js

```diff
@@ -85,7 +85,7 @@
                         function () {
                             $.ajax({
                                 type: 'DELETE',
-                                url: self.teamMembershipDetailUrl + username
+                                url: self.teamMembershipDetailUrl.concat(username, '?admin=true')
```

Comment:
Yeah, I'm entirely fine with the querystring approach. My earlier was just about passing the data to jQuery in a different way, but I didn't mean to change which part of the request holds the data.